com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md

```diff
@@ -40,7 +40,8 @@ When you create an HDRP Asset, open it in the Inspector to edit its properties.
 | **Lit Shader Mode**                     | Use the drop-down to choose which mode HDRP uses for the [Lit Shader](Lit-Shader.md).<br />&#8226; **Forward Only**: forces HDRP to only use forward rendering for Lit Shaders.<br />&#8226; **Deferred Only**: forces HDRP to use deferred rendering for Lit Shaders (HDRP still renders advanced Materials using forward rendering).<br />&#8226; **Both**: allows the Camera to use deferred and forward rendering.<br /><br />Select **Both** to allow you to switch between forward and deferred rendering for Lit Shaders at runtime per Camera. Selecting a specific mode reduces build time and Shader memory because HDRP requires less Shader variants, but it is not possible to switch from one mode to the other at runtime. |
 | **- Multisample Anti-aliasing Quality** | Use the drop-down to set the number of samples HDRP uses for multisample anti-aliasing (MSAA). The larger the sample count, the better the quality. Select **None** to disable MSAA.<br />This property is only visible when **Lit Shader Mode** is set to **Forward Only** or **Both**. |
 | **Motion Vectors**                      | Enable the checkbox to make HDRP support motion vectors. HDRP uses motion vectors for effects like screen space reflection (SSR) and motion blur. When disabled, motion blur has no effect and HDRP calculates SSR with lower quality. |
-| **Runtime Debug Display**               | Enable the checkbox to make HDRP display Material and Lighting properties at runtime to help debugging.Disable this checkbox to reduce build time and Shader memory. This disables the following debug modes: All Material debug modes except GBuffer debug. The Lux meter, diffuse lighting only, and specular lighting only debug modes. The overriding option for overriding albedo. |
+| **Runtime Debug Display**               | Enable the checkbox to make HDRP being able to use debug mode from RenderPipeline windows at runtime. Disable this checkbox to reduce build time and Shader memory. This disables the following debug modes: All material properties debug modes except GBuffer debug. The various properties overrides options. All the lighting debug modes.  |
+| **Runtime AOV API**                     | Enable the checkbox to make HDRP being able to use AOV API (Rendering of Material properties an lighting mode) at runtime. Disable this checkbox to reduce build time and Shader memory. This disables the following AOV modes: All material properties and Lighting modes. |
```

Comment:
Enable the checkbox to make HDRP able to use the AOV API (rendering of material properties and lighting modes) at runtime. Disable this checkbox to reduce build time and shader memory. This disables the following AOV modes: All material properties and lighting modes.